### ISSUE TITLE
Provide more detailed country information

### ISSUE THREAD

**Issue by axelboc** [axelboc/anki-ultimate-geography#13]
Discussion started in issue #11.

For non-sovereign states and, more generally, cards that are not easy to memorise, provide additional country information in the form of interesting, unique **historical and geographical facts**. The facts should be sourced from Wikipedia and written in such a way that they are unlikely to become outdated (if needed, specify the date the fact was added).

**Comment by axelboc** [comment]
@code-hunger so you're feeling up for it? Do you want to open a PR and commit to that as you go? I think we're on the same page but let's keep each info as short as possible. I haven't used the deck for quite a while, so I won't be contributing anything myself, but happy to help if you're not sure about which facts to pick or whatever.

**Comment by p-hofer** [comment]
I just found your deck - and love it! I'd be more than happy to help contributing. If I can't remember a country I'll try to find a nice fact about it. @axelboc What's the way to go here? Should I create my own PR and push facts in there?

**Comment by axelboc** [comment]
Great, thanks! Hmm perhaps create an issue each time -- then we can nail down the wording and ask @code-hunger to add it to #14. How does that sound?

**Comment by code-hunger** [comment]
Hi, and sorry for the long delay. I just pushed my changes so far (mainly flag similarity hints, as they were the most difficult to learn), but there's still stuff to do.
There were some conflicts with upstream/master so I did a rebase (in case you encounter some problems pulling my branch).

**Comment by axelboc** [comment]
@code-hunger @OnkelTem - Following up on your discussion #51 about adding extra info to cards to help with memorisation :

I think we all agree that **historical and cultural facts are a no-go**. From a contributor's point of view, it's tedious to add to the deck : there's too much choice each time; the choice is very subjective; and it's difficult to write facts in a concise and consistent way. From a user's point of view, it's most likely to offend : there'll always be someone who thinks that a history fact itself is biased (the old saying that history is written by the winners) or that the choice of the fact is biased; facts can overlap with stereotypes; and reducing a country's history and culture to one or two facts is, in a way, quite demeaning.

I think it's also clear that GDP and population size info would be impossible to maintain and would not really help with memorisation (lots of similar numbers). I'd argue that languages wouldn't either, since many countries share the same languages and I'm sure many others have obscure languages that would be of little help with remembering a card. Feels more like a field to me than extra info on a card.

Last but not least, I believe that not all cards need extra info, far from it. The most difficult cards to remember are those of islands, micro-nations, small dependent territories (which are mostly islands) and newly-sovereign states (e.g. Artsakh). The sovereignty info is already of some help, but these cards could definitely benefit from the following improvements:

- Most islands could really use some **more precise maps**.
- All of them could use some very simple, unequivocal, never-changing and easy-to-add **geographical facts** that fit in one or two short sentences -- for instance:
  - whether it's a micro-nation, principality, etc. (yup, that's not even mentioned);
  - whether it borders any countries;
  - for islands, the type of landmass (atoll, archipelago, etc.)
  - whether an island is in the Caribbean, the Pacific, etc. (which is a big hint, since Caribbean islands tend to have capitals with more European-sounding names, for example);
  - whether it's way out in the middle of the Pacific, or not far off the coast of some country,
  - whether it's part of a famous group of islands (e.g. the ABC islands) or sub-region.

These two improvements already have their own issues: #1 and #60. What do you think? Do you feel like the above would sufficiently help with memorisation? Have I missed something else that is difficult to learn? Similar flags are still a problem, perhaps there's something more to do there as well, like add a hint to help with the most tricky ones (I'm thinking of the pair Galapagos // Sierra Leone, which blues are just too similar -- perhaps a hint like "Not Sierra Leone" 😄).

**Comment by code-hunger** [comment]
> - whether an island is in the Caribbean, the Pacific, etc. (which is a big hint, since Caribbean islands tend to have capitals with more European-sounding names, for example);
> - whether it's way out in the middle of the Pacific, or not far off the coast of some country

That can be already seen on the map, and I think it's best not to reveal any location information. I want to be able to instantly determine the location of "Madeira" with a relative accuracy when prompted with that name.  
If I'm presented with __Madeira - Portugese archipelago near the Atlantic shore of North Africa__ this just won't work. Even if it is shown only on the Flag-Country card, it will remind me of its location, and the next day when I have to determine the country on the map, I'll be able to do so easily because I was just reminded that. Or maybe it's subjective and I'm becoming too obsessed and we shouldn't take things so seriously :)

**Comment by axelboc** [comment]
Fair point. I feel like it sill leaves perhaps the landmass and, in some cases, the general area (i.e. Pacific, Caribbean). In the case of Mediterranean/Atlantic islands especially -- since there aren't many of them -- I reckon the landmass would suffice.

**Comment by mighty-cthulhu** [comment]
I think more precise maps should do the trick. However if you do decide to add more info, please display on the back of the card, not the front.

**Comment by axelboc** [comment]
Does anyone think that this issue would still be relevant if all the maps were sufficiently precise?

**Comment by aplaice** [comment]
I think that this issue and detailed maps are mostly orthogonal — the idea behind this issue seems to be adding some "emotional salience" to the countries and even detailed maps don't help much with that. However, given the unworkability of the issue — the extra information would be at least one of: subjective, excessively detailed or hard-to-maintain — it's probably safe to close.

<hr/>

Crazy idea: once ignorable fields (https://github.com/Stvad/CrowdAnki/issues/62) are implemented, we could add an empty-by-default, non-exportable, personal, extra-info field, visible on the back side. Learners would be (somehow™\*) encouraged to add some interesting information when they come across an unfamiliar country/region, if they want to. The purpose would be to:

1. encourage them to read up on previously unknown states, since as is often pointed out (and as I frequently ignore...) Anki is not really suitable for learning new material, but for not forgetting already learned material,

2. provide a personalised "extra info" field which would be interesting for the given person.

\* in the deck description? as a faint, tiny-font, opt-in (i.e. by default commented-out) nag text with a link to the Wikipedia page (AFAIR Wikipedia links can be directly generated from the country name)?

(Basically it's an adaptation of fluent forever's idea of people finding their own images for their own decks.)
